app/api/bread/views.py

```diff
@@ -0,0 +1,100 @@
+import datetime
+
+import flask_jwt_extended as jwt
+from flask import request
+from flask_restful import Resource
+
+from app import db
+from app.api import api
+from app.models import BreadOrder, BreadType
+from .schema import OrderSchema, BreadListSchema, BreadTypeSchema
+
+order_schema = OrderSchema(many=True)
+breadlist_schema = BreadListSchema()
+bread_types_schema = BreadTypeSchema(many=True)
+
+
+@api.resource('/bread/')
+class BreadOrderListResource(Resource):
+
+    @jwt.jwt_required
+    def get(self):
+        startdate = datetime.date.today()
+        if startdate.month >= 9:
+            startdate = startdate.replace(month=9, day=1)
+        else:
+            startdate = startdate.replace(month=1, day=1)
```

Comment:
waarom de stardate replacen?

Reply:
Dat zijn gewoon alle weken van het semester, en op deze manier is het jaar altijd correct.

Reply:
Dus wat wil je nu als startdag hebben? Deze dag? De 1e dag van het academiejaar? Das echt niet duidelijk voor mij. Kan de startdag niet gewoon deze dag op de server zijn? Wat moet er juist teruggeven worden? Alle brood reservaties van het hele jaar? Want mijn idee is wel om elk jaar met een nieuwe database te starten ...

Reply:
Ik wil dat mensen nog kunnen zien wat ze besteld hebben tijdens het semster. Op deze manier zijn gewoon alle entries van de voorbije weken ook te zien, maar niet van vorige semsters.

Reply:
Oke, een history van broden lijkt mij goed. Ik vind het gewoon wat lelijk om zo te doen ... Als het nu werkt is het ok, maar misschien interessant voor later is een history button die dan gewoon alle reservaties van jou in het verleden toont? Ipv bij de bestellingen pagina?